src/Api/PubnubApi/EndPoint/Presence/LeaveOperation.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PubnubApi.EndPoint
+{
+	public class LeaveOperation : PubnubCoreBase
+	{
+		private readonly PNConfiguration config;
+		private readonly IJsonPluggableLibrary jsonLibrary;
+		private readonly IPubnubUnitTest unit;
+		private readonly IPubnubLog pubnubLog;
+		private readonly TelemetryManager pubnubTelemetryManager;
+
+		public LeaveOperation(PNConfiguration pubnubConfig, IJsonPluggableLibrary jsonPluggableLibrary, IPubnubUnitTest pubnubUnit, IPubnubLog log, TelemetryManager telemetryManager, EndPoint.TokenManager tokenManager, Pubnub instance) : base(pubnubConfig, jsonPluggableLibrary, pubnubUnit, log, telemetryManager, tokenManager, instance)
+		{
+			config = pubnubConfig;
+			jsonLibrary = jsonPluggableLibrary;
+			unit = pubnubUnit;
+			pubnubLog = log;
+			pubnubTelemetryManager = telemetryManager;
+
```

Comment:
as above - Why is that whitespace here? Does it improve readability for you? If yes - you can leave it.

Reply:
fixed!